components/Cards/src/ColorThemer/MDCCardsColorThemer.m

```diff
@@ -28,4 +32,24 @@ + (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
   cardCell.backgroundColor = colorScheme.surfaceColor;
 }
 
++ (void)applyStrokeVariantWithColorScheme:(nonnull id<MDCColorScheming>)colorScheme
+                                   toCard:(nonnull MDCCard *)card {
+  card.backgroundColor = colorScheme.surfaceColor;
+  [card setShadowElevation:kStrokeVariantElevation forState:UIControlStateNormal];
+  [card setBorderWidth:kStrokeVariantBorderWidth forState:UIControlStateNormal];
+  UIColor *borderColor =
+      [colorScheme.onSurfaceColor colorWithAlphaComponent:kStrokeVariantBorderOpacity];
+  [card setBorderColor:borderColor forState:UIControlStateNormal];
+}
+
++ (void)applyStrokeVariantWithColorScheme:(id<MDCColorScheming>)colorScheme
+                               toCardCell:(MDCCardCollectionCell *)cardCell {
+  cardCell.backgroundColor = colorScheme.surfaceColor;
```

Comment:
we should clean the other states before applying the theme, so the other states will fallback to Normal. Please see what we did in the button themer